File: ueueo-framework/ueueo-blob-storing/src/main/java/Abp/BlobStoring/BlobAlreadyExistsException.cs
using System;
using System.Runtime.Serialization;

namespace Volo.Abp.BlobStoring;

public class BlobAlreadyExistsException : AbpException
{
    public BlobAlreadyExistsException()
    {

    }

    public BlobAlreadyExistsException(String message)
        : base(message)
    {

    }

    public BlobAlreadyExistsException(String message, Exception innerException)
        : base(message, innerException)
    {

    }

    public BlobAlreadyExistsException(SerializationInfo serializationInfo, StreamingContext context)
        : base(serializationInfo, context)
    {

    }
}

File: abp-frameworks/ueueo-blob-storing/src/main/java/com/ueueo/blobstoring/BlobAlreadyExistsException.java
package com.ueueo.blobstoring;

import com.ueueo.exception.BaseException;

public class BlobAlreadyExistsException extends BaseException {
    public BlobAlreadyExistsException() {

    }

    public BlobAlreadyExistsException(String message) {
        super(message);
    }

    public BlobAlreadyExistsException(String message, Exception innerException) {
        super(message, innerException);
    }

}
